app/ui-react/packages/ui/src/Data/Virtualizations/VirtualizationList.tsx

```diff
@@ -68,6 +68,7 @@ export class VirtualizationList extends React.Component<
     return (
       <>
         <SimplePageHeader
+          className="container-fluid pf-u-my-md"
```

Comment:
Would it make sense to add this to the SimplePageHeader component to minimize the amount of custom classes added to these kinds of elements?